apps/src/sites/code.org/pages/views/analytics_event_log_helper.js

```diff
@@ -0,0 +1,9 @@
+import $ from 'jquery';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+
+// Log page visits on document.ready.
+$(document).ready(() => {
+  const eventLogger = document.getElementById('analytics-event-log-helper');
+  const pageVisitEventName = eventLogger?.dataset?.eventName;
+  analyticsReporter.sendEvent(pageVisitEventName);
```

Comment:
actually I do have one more question here which @bethanyaconnor may have an opinion on -- I'm worried that we could lose some events if we accidentally pass an empty value here, and if we do it will be hard to figure out which line of code the blanks are coming from. what do you think about modifying `analyticsReporter.sendEvent` to throw and exception if the event name is falsy? (definitely not in this PR, but a question for the benefit of future amplitude logging)

Reply:
Oh interesting! Part of me doesn't like the idea of logging possibly crashing the page but the other part of me agrees that it would be tough to track down missing events. Could we send an error to NewRelic or something instead?

Reply:
Sounds good! I'll add a NewRelic logToCloud call! Is there anything I need to do on the NewRelic side or just create the call in our code base?

Reply:
Thank you for the feedback! I'm going to move this error handling into a separate PR to have more focused attention and so the preliminary work can get sent through!